front/src/components/Login.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+
+export function Login() {
+    const [userName, setId] = useState('');
```

Comment:
```suggestion
    const [username, setId] = useState('');
```

何故かは知らないけど慣用的にusernameは1単語として表記することが多い